src/components/MenuAppBar/MenuAppBar.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect } from 'react';
+import Toolbar from '@mui/material/Toolbar';
+import IconButton from '@mui/material/IconButton';
+import AccountCircle from '@mui/icons-material/AccountCircle';
+import MusicVideoIcon from '@mui/icons-material/MusicVideo';
+import { useNavigate, useLocation } from 'react-router-dom';
+
+import { Pages } from '../../constants';
+import {
+  StyledBox,
+  StyledAppBar,
+  StyledTypography,
+  StyledIconButton,
+} from './styles';
+
+const MenuAppBar = () => {
+  const navigate = useNavigate();
+  const location = useLocation();
+
+  const [isActive, setIsActive] = useState({
+    feed: false,
+    profile: false,
+  });
+
+  useEffect(() => {
+    switch (location.pathname) {
+      case Pages.Feed:
+        setIsActive({ feed: true, profile: false });
+        break;
+      case Pages.Profile:
+        setIsActive({ feed: false, profile: true });
+        break;
+      default:
+        setIsActive({ feed: false, profile: false });
+        break;
+    }
+  }, [location]);
```

Comment:
when you use dependency for the whole object 'location', rerender will provide after each change of this object. 
so, if you later will add some changes to location logic, it will rerender the component again. So probably better to use only the exact "location.pathname" as a dependency